conf/machine/olinuxino-a20lime2-emmc.conf

```diff
@@ -0,0 +1,12 @@
+#@TYPE: Machine
+#@NAME: Olimex A20-OLinuXino Lime2 Board with 4GB eMMC
+#@DESCRIPTION: Machine configuration for the Olimex A20-OLinuXino Lime2 Board with 4GB eMMC, based on Allwinner A20 CPU
+#https://github.com/OLIMEX/OLINUXINO
+
+require conf/machine/include/sun7i.inc
+
+PREFERRED_VERSION_linux = "4.11.0+git%"
```

Comment:
Sorry for the delay, if it's not too much of a hassle please backport the dts fix and i will merge this.
Thanks!